fastchat/serve/rwkv_model.py

```diff
@@ -0,0 +1,30 @@
+import os
+from types import SimpleNamespace
+import warnings
+
+import torch
+
+os.environ["RWKV_JIT_ON"] = '1'
+os.environ["RWKV_CUDA_ON"] = '1'
+
+from rwkv.model import RWKV
+from rwkv.utils import PIPELINE, PIPELINE_ARGS
+
+
+class RwkvModel:
+    def __init__(self, model_path):
+        warnings.warn("Experimental support. Please use ChatRWKV if you want to chat with RWKV")
```

Comment:
What does this warning mean? Can I use this model for the chatbot arena?

Reply:
@BlinkDL what is the meaning of this warning? Can we use the this implementation for chatting?